common/autoware_universe_utils/src/ros/diagnostics_module.cpp

```diff
@@ -76,7 +76,7 @@ void DiagnosticsModule::add_key_value(const std::string & key, const bool & valu
 
 void DiagnosticsModule::update_level_and_message(const int8_t level, const std::string & message)
 {
-  if ((level > diagnostic_msgs::msg::DiagnosticStatus::OK)) {
+  if ((level >= diagnostic_msgs::msg::DiagnosticStatus::OK)) {
```

Comment:
@YamatoAndo 
Is this change consistent with your original intent?

Reply:
no

Reply:
If you want to do so, this if statement will always be satisfied and is unnecessary, so please remove it.
Additionally, ensure that "; " is not appended when the message is empty.

Also, it would be better to revise the code at line 102.
like this,
```
  if (diagnostics_msg.status.at(0).message.empty()) {
    if (diagnostics_msg.status.at(0).level == diagnostic_msgs::msg::DiagnosticStatus::OK) {
    diagnostics_msg.status.at(0).message = "OK";
    }
    else if (diagnostics_msg.status.at(0).level == diagnostic_msgs::msg::DiagnosticStatus::WARN) {
    diagnostics_msg.status.at(0).message = "WARN";
    }
    else if (diagnostics_msg.status.at(0).level == diagnostic_msgs::msg::DiagnosticStatus::ERROR) {
    diagnostics_msg.status.at(0).message = "ERROR";
    }
    else if (diagnostics_msg.status.at(0).level == diagnostic_msgs::msg::DiagnosticStatus::STALE) {
    diagnostics_msg.status.at(0).message = "STALE";
    }
  }
```

Reply:
Sorry that was a complete mistake that I introduced while I played around with the code. Thanks for the catch. Fixed: https://github.com/autowarefoundation/autoware.universe/pull/9714/commits/113d770fb442309642d869971a78472eda2d005c